src/pudl_archiver/depositors/zenodo.py

```diff
@@ -117,6 +118,7 @@ async def run_request():
                     asyncio.TimeoutError,
                     ZenodoClientError,
                 ),
+                retry_count=retry_count,
```

Comment:
Just confirming, you're basically passing the option here to lower the default retry count from 7 to 5 for deletions? What's the motivation for doing this?

Reply:
The motivation is impatience - it seems to typically resolve within 5 retries, so I don't want to wait an extra 90 seconds while waiting for the deletion to register.

Reply:
Fair enough!